frontend/dockerfile/dockerfile_test.go

```diff
@@ -7847,10 +7948,16 @@ func testHistoryFinalizeTrace(t *testing.T, sb integration.Sandbox) {
 
 	f := getFrontend(t, sb)
 
-	dockerfile := []byte(`
+	dockerfile := []byte(integration.UnixOrWindows(
+		`
 FROM scratch
 COPY Dockerfile /foo
-`)
+`,
+		`
+FROM nanoserver
```

Comment:
Out of scope for this PR: Is the `FROM scratch` not working at all in windows? I think in buildkit we should make it work. It should fail if you need to create image/container on top of `FROM scratch` stage, but if you just use `docker buildx build -o out --target=binaries .` then that should work in windows as well just to get the artifacts without base image.

Reply:
ok, opened a tracking issue here - https://github.com/moby/buildkit/issues/5264